lib/builderator/tasks/vagrant.rb

```diff
@@ -98,13 +98,6 @@ def destroy(profile = :default, *args)
         end
       end
 
-      desc 'rebuild ARGS', 'Destroy and recreate Vagrant VM(s)'
-      method_option :force, :aliases => :f, :type => :boolean, :default => true
-      def rebuild(profile = :default, *args)
-        destroy(profile, *args)
-        up(profile, *args)
-      end
```

Comment:
The way we select what Vagrant provider to use needs to be made more generic. ATM there isn't really a way to have `rebuild` choose between `local` and `ec2